### ISSUE TITLE
Hiden scores display as 1 pt

### ISSUE THREAD

**Issue by OMGTallMonster** [michael-lazar/rtv#150]
Some subreddits choose to [hide the scores of new comments](https://www.reddit.com/r/modnews/comments/1dd0xw/moderators_new_subreddit_feature_comment_scores/) for a period of time. PRAW Comment objects contain a `score_hidden` attribute that is set to True when this is the case.

It would probably be sufficient to add a small conditional in `content.py` at line 91:

```
            if comment.score_hidden:
                data['score'] = '[score hidden]'
            else:
                data['score'] = '{} pts'.format(comment.score)
```

There is also a `hide_score` attribute on PRAW Submission objects, but don't know if there should be similar logic for submissions. I don't recall ever seeing a submission with a hidden score, and the PRAW documenation seems to be sparse.

**Comment by michael-lazar** [comment]
Nice catch. Definitely sounds like it's worth adding, feel free to make a pull request :wink: 

Throwing out some other possibilities for formatting.
```
civilization_phaze_3 • [score hidden] 4hr
civilization_phaze_3 • [hidden] 4hr
civilization_phaze_3 • [?] 4hr
civilization_phaze_3 • � 4hr
civilization_phaze_3 • ? pts 4hr
civilization_phaze_3 • - pts 4hr
civilization_phaze_3 • - 4hr
civilization_phaze_3 • 4hr
```

I looked around for documentation on ``hide_score`` but couldn't find anything either. I think it's for when a submission is new and doesn't have any votes yet? Like how the top post here doesn't have a score yet.
![selection_004](https://cloud.githubusercontent.com/assets/5026795/10158624/34e68aec-6648-11e5-8441-068e0d6382e6.png)
I tested a few out and was able to verify that ``hide_score`` was actually getting set to true, with ``ups=1`` and ``score=1``.
